Kernel/Devices/Audio/Management.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Singleton.h>
+#include <Kernel/Bus/PCI/API.h>
+#include <Kernel/Bus/PCI/IDs.h>
+#include <Kernel/Devices/Audio/AC97.h>
+#include <Kernel/Devices/Audio/Management.h>
+#include <Kernel/Devices/Audio/SB16.h>
+#include <Kernel/Sections.h>
+
+namespace Kernel {
+
+static Singleton<AudioManagement> s_the;
+static Atomic<u32> s_device_minor_number;
+
+MajorNumber AudioManagement::audio_type_major_number()
+{
+    return 116;
```

Comment:
Does this have a specific reason for being 116 and not 42 like before?

Reply:
I took this from Linux, so I did `ls -la /dev/snd` on my machine and saw that major number.
I don't think it matters really but I won't insist on keeping it 116.

Reply:
From discord server #kernel channel:

> 1:19 PM] supercomputer7: @Filmröllchen, Sampler of Audio I guess we are fine keeping the number 116 for sound devices?
[1:19 PM] Filmröllchen, Sampler of Audio: yup